### DISCUSSION TITLE
After last update dev Version - a lot of things are not working

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5195] [General]
Hi dev team,
 

after update to c8815b8 a lot of things are not working. Some things listed:

- whole EFIS out of order. Nothing will be displayed on the ND. When you push the button no green lights. The different modes are not working nore the range adjustment.
- If I rotate captains side knobs, copilots turns as well :-p ....
- ECAM out of order. Only fuel page is selectable.
- preselected altitude is 100. If you will select for example 5000 you got 5100 and have to change this with the option on the knob.
- FD not working ...

too bad, not flyable in this stage and I have no backup from a prior version ... ahrg :-p ... good, can swith to experimental ... :-)

I know you are still in development mode, but why is it sometimes the case that working things are inop after an update? 

Greets Klaus

**Comment by Benjozork** [comment]
This is an isolated issue and we have not heard any other reports of that. Please make sure you do a clean reinstall, as that issue is local.

**Reply by CptBayer** [reply]
Hi thanx for the quick reply....

I can try this. Just for information, I always use your newest installer and I also perform the updates via the installer.

Greets Klaus

**Comment by CptBayer** [comment]
02cf051 is the displayed version after clean new instalation .... just for info ...

**Reply by CptBayer** [reply]
working :-) ... Don´t ask what happend.... Next time I´ll try a clean install prior reporting :-p ...

Greets Klaus
